lib/matplotlib/animation.py

```diff
@@ -43,7 +44,7 @@
 
 # Process creation flag for subprocess to prevent it raising a terminal
 # window. See for example:
-# https://stackoverflow.com/questions/24130623/using-python-subprocess-popen-cant-prevent-exe-stopped-working-prompt
+# https://stackoverflow.com/q/24130623/353337
```

Comment:
I don't know why you get this trailing stuff
```suggestion
# https://stackoverflow.com/q/24130623
```

Reply:
My bad, this is what you get when you press "Share" below a question. (It includes the user ID then.) Removing all of those.